Patch patchlevel.patch for the patch queue to patch the patchlevel.

diff -r 059a1d81b9fb include/patchlevel.h
--- a/include/patchlevel.h
+++ b/include/patchlevel.h
@@ -18,7 +18,7 @@
  * $Id: patchlevel.h,v 1.23.2.22 2007/03/09 23:49:31 entrope Exp $
  *
  */
-#define PATCHLEVEL "10+snircd(1.3.4a)"
+#define PATCHLEVEL "10+snircd(1.3.5dev)"
 
 #define RELEASE ".12."
 
